modules/rostests/apitests/apphelp/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 project(appcompat)
-add_definitions(-D__ROS_LONG64__ -DWINETEST_USE_DBGSTR_LONGLONG)
+add_definitions(-D__ROS_LONG64__ -DWINETEST_USE_DBGSTR_LONGLONG -DUNICODE -D_UNICODE)
```

Comment:
The UNICODE flag to set_module_type is usually preferable.

Reply:
It also allows our CMake macros to select the correct entry-point name, amongst others...

Reply:
It would be, were it not for the fact that `<wine/test.h>` in `STANDALONE` mode does not define a unicode entrypoint.